src/it/it-source-failed/src/test/java/name/valery1707/kaitai/it/KaitaiIcoTest.java

```diff
@@ -0,0 +1,49 @@
+package name.valery1707.kaitai.it;
+
+import io.kaitai.struct.ByteBufferKaitaiStream;
+import io.kaitai.struct.KaitaiStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KaitaiIcoTest {
+	private KaitaiStream getResourceAsStream(String resource) {
+		try (
+			InputStream in = getClass().getResourceAsStream(resource);
+			ByteArrayOutputStream out = new ByteArrayOutputStream();
+		) {
+			byte[] buffer = new byte[1024 * 8];
+			int len;
+			while ((len = in.read(buffer)) > 0) {
+				out.write(buffer, 0, len);
+			}
+			out.flush();
+			return new ByteBufferKaitaiStream(out.toByteArray());
```

Comment:
\[Checkstyle\] WARNING: 'try' child has incorrect indentation level 24, expected level should be 6\.